examples/deadline/All-In-AWS-Infrastructure-SEP/python/setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+
+setuptools.setup(
+    name="all_in_aws_infrastructure_sep",
+    version="0.0.1",
+
+    description="RFDK All In AWS Infrastructure SEP",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+
+    package_dir={"": "package"},
+    packages=setuptools.find_packages(where="package"),
+
+    install_requires=[
+        "aws-cdk.core==1.70.0",
+        "aws-rfdk==0.19.0"
```

Comment:
We're on CDK 1.72.0 and RFDK 1.20.0 now. Needs to be updated here and in the ts example's package.json.